exercises/pythagorean-triplet/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "pythagorean_triplet"
+version = "0.0.0"
```

Comment:
there is no canonical JSON in https://github.com/exercism/problem-specifications/tree/master/exercises/pythagorean-triplet , so it couldn't have been called 1.0.0, so I put it back at 0.